app/views/tasks/replace.html.haml

```diff
@@ -0,0 +1,85 @@
+.govuk-grid-row
+  .govuk-grid-column-full
+    = link_to 'Back', task_path(@task), { class: 'govuk-back-link', title: 'Back to the task page' }
+
+.govuk-grid-row
+  .govuk-grid-column-full
+    %h1.govuk-heading-xl
+      Correction
+
+    %dl.govuk-summary-list.govuk-summary-list--no-border
+      .govuk-summary-list__row
+        %dt.govuk-summary-list__key
+          Framework
+        %dd.govuk-summary-list__value
+          = @task.framework.short_name
+          = @task.framework.name
+
+      .govuk-summary-list__row
+        %dt.govuk-summary-list__key
+          Month
+        %dd.govuk-summary-list__value
+          = @task.reporting_period
+
+      .govuk-summary-list__row
+        %dt.govuk-summary-list__key
+          Reported
+        %dd.govuk-summary-list__value
+          = @task.completed_at
+
+      - if current_user.multiple_suppliers?
+        .govuk-summary-list__row
+          %dt.govuk-summary-list__key
+            Supplier
+          %dd.govuk-summary-list__value
+            = @task.supplier_name
+
+    %p
+      You will need to pay the management charge from the incorrect return unless a successful corrected return is made.
+
+    - unless @task.latest_submission.report_no_business?
+      .govuk-grid-row
+        .govuk-grid-column-full
+          = link_to 'Report no business', new_task_no_business_path(@task, replacement: true), class: 'govuk-button'
+
+    %h2.govuk-heading-m
+      Incorrect return
+
+    - if @submission.report_no_business?
+      %p
+        You reported no business
+    - else
+      %dl.govuk-summary-list.govuk-summary-list--no-border
+        .govuk-summary-list__row
+          %dt.govuk-summary-list__key
+            Invoiced
```

Comment:
Given that we don't yet have the data to tell the user if they've been invoiced/credited, and as was demonstrated in the user research playback they confused users anyway, we should skip the "Invoiced" and "Credit note" fields for now.